### ISSUE TITLE
Update Deletion in MODIF

### ISSUE THREAD

**Issue by ChriD** [ChriD/SAGEX3#1]
The TABLEAUX script handles the deletion better. It stores the original line nums into an array and on MODIF it checks if those line nums exists in the screen or not (if they do not exists they will be deleted)
--> Find a way to improve the deletion method of this script and keep the ability to use it for multiple details!

**Comment by mcNets** [comment]
Hi ChriD, nice script, is this project still active?  
Have you the source of TABLEAUX scripts?

**Comment by ChriD** [comment]
Not really. I uesed it in some projects but i did not have any X3 projects the last few years. (i have to admit that X3/Safe X3 was not my favorite programm/coding language i had to develop something in)
I do indeed have the source code of the TABLEAUX script for old PU9PL2 version.of X3 but i think i am not allowed to post it here.

**Comment by mcNets** [comment]
Neither my favourite one, but my company has acquired X3 last year. In case you could share it, my mail is mcnets@gmail.com
